ckanext/data_qld/fanstatic/flag-comment.js

```diff
@@ -91,7 +91,7 @@ this.ckan.module('flag-comment', function (jQuery) {
       event.preventDefault();
       var comment_id = this.options.comment_id;
       var element = this
-      jQuery.post('/comment/' + comment_id + '/flag', function() {
```

Comment:
so back to get?
this is not side effect free so it should be a post, were you unable to add the csrf element's required for this?
https://medium.com/@robert.broeckelmann/http-post-vs-get-is-one-more-secure-for-use-in-rest-apis-2469753121b0

Reply:
It could be done, but a) it would require more rework to reliably extract the token and attach it, b) it wouldn't achieve anything unless we also modified `ckanext-ytp-comments` to reject GET requests, and c) it's not a regression, so it would be good to first get this release out.